examples/tutorials/asr_inference_with_ctc_decoder_tutorial.py

```diff
@@ -222,28 +224,212 @@ def forward(self, emission: torch.Tensor) -> str:
 #
 # Now that we have the data, acoustic model, and decoder, we can perform
 # inference. Recall the transcript corresponding to the waveform is
-# ``"when it was the seven hundred and eighteenth night"``
+# ``"i really was very much afraid of showing him how much shocked i was at some parts of what he said"``
 #
 
+actual_transcript = "i really was very much afraid of showing him how much shocked i was at some parts of what he said"
```

Comment:
This variable is always used as `actual_transcript.split()`, so I think you can `split()` here.